apps/meteor/client/views/admin/integrations/outgoing/OutgoingWebhookForm.tsx

```diff
@@ -476,7 +476,27 @@ const OutgoingWebhookForm = () => {
 								<Controller
 									name='retryCount'
 									control={control}
-									render={({ field }) => <TextInput id={retryCountField} {...field} aria-describedby={`${retryCountField}-hint`} />}
+									render={({ field }) => (
+										<TextInput
+											{...field}
+											id={retryCountField}
+											type='number'
+											min='0'
+											step='1'
+											// This 'onChange' is the new magic
```

Comment:
Please don't leave comments on the component, usually we only use comments to explain code that might be counter intuitive or confusing